tfhe/src/c_api/core_crypto/mod.rs

```diff
@@ -0,0 +1,128 @@
+use super::utils::*;
+use std::os::raw::c_int;
+
+#[no_mangle]
+pub unsafe extern "C" fn core_crypto_lwe_multi_bit_bootstrapping_key_element_size(
+    input_lwe_sk_dim: usize,
+    output_glwe_sk_dim: usize,
+    output_glwe_sk_poly_size: usize,
+    lwe_multi_bit_base_log: usize,
+    lwe_multi_bit_level_count: usize,
+    lwe_multi_bit_grouping_factor: usize,
+    result: *mut usize,
+) -> c_int {
+    catch_panic(|| {
+        use crate::core_crypto::prelude::*;
+
+        let result = get_mut_checked(result).unwrap();
+
+        let input_lwe_sk_dim = LweDimension(input_lwe_sk_dim);
+
+        let output_glwe_sk_dim = GlweDimension(output_glwe_sk_dim);
+        let output_glwe_sk_poly_size = PolynomialSize(output_glwe_sk_poly_size);
+
+        let lwe_multi_bit_base_log = DecompositionBaseLog(lwe_multi_bit_base_log);
+        let lwe_multi_bit_level_count = DecompositionLevelCount(lwe_multi_bit_level_count);
+        let lwe_multi_bit_grouping_factor = LweBskGroupingFactor(lwe_multi_bit_grouping_factor);
+
+        let bsk = LweMultiBitBootstrapKeyOwned::new(
+            0u64,
+            output_glwe_sk_dim.to_glwe_size(),
+            output_glwe_sk_poly_size,
+            lwe_multi_bit_base_log,
+            lwe_multi_bit_level_count,
+            input_lwe_sk_dim,
+            lwe_multi_bit_grouping_factor,
+            CiphertextModulus::new_native(),
+        );
+
+        *result = bsk.into_container().len();
```

Comment:
can definitely avoid an allocation here, it was done quick and dirty (yikes)

Reply:
multi-bit pbs key has this size:

```
int multi_bit_bsk_size = lwe_dimension * pbs_level * (glwe_dimension + 1) *
                 (glwe_dimension + 1) * polynomial_size *
                 (1 << grouping_factor) / grouping_factor;
```

Reply:
yeah, I'll re-use some primitives from concrete-core

Reply:
@agnesLeroy be careful there is an API break, if you can just check the new entry point works ok for you ?

Reply:
the formula looks good on your end anyways, but you can use the entry point from the C API to have consistency if you want